package.json

```diff
@@ -19,6 +19,7 @@
   "homepage": "https://github.com/Harvard-ATG/scalar-integration-tests#readme",
   "dependencies": {},
   "devDependencies": {
+    "@cypress/skip-test": "^2.6.1",
```

Comment:
Nice find on this library. It's a shame that cypress doesn't support this functionality natively.

Reply:
I think there is a skip function but it doesn't accept parameters like this library does.

Reply:
Ah good to know!